src/runtime_src/xdp/profile/device/hal_device/xdp_hal_device.cpp

```diff
@@ -155,12 +155,12 @@ uint64_t HalDevice::getDeviceAddr(size_t id)
 
 double HalDevice::getMaxBwRead()
 {
-  return xclGetReadMaxBandwidthMBps(mHalDevice);
+  return 9600.0;
```

Comment:
We can define xclGetReadMaxBandwidthMBps, xclGetWriteMaxBandwidthMBps in the Edge HAL. That will help if PCIe and Edge flow have different bandwidth. I have the change in my local and plan to open Pull request with that soon. So, this current change can go in now.